src/Services/Orders/Distribt.Services.Orders/Program.cs

```diff
@@ -4,19 +4,19 @@
 using Distribt.Services.Orders.Data;
 using Distribt.Services.Orders.Services;
 
-WebApplication app = DefaultDistribtWebApplication.Create(args, webappBuilder =>
+WebApplication app = await DefaultDistribtWebApplication.Create(args, async webappBuilder =>
 {
     MongoMapping.RegisterClasses();
     webappBuilder.Services.AddServiceBusDomainPublisher(webappBuilder.Configuration);
-    webappBuilder.Services.AddDistribtMongoDbConnectionProvider(webappBuilder.Configuration);
+    await webappBuilder.Services.AddDistribtMongoDbConnectionProvider(webappBuilder.Configuration);
```

Comment:
you dont need this here; as this is inside add product service

Reply:
done